promitor-agent-resource-discovery/README.md

```diff
@@ -81,12 +81,23 @@ their default values.
 | `health.readiness.enabled`  | Indication if readiness probes should be used | `true`            |   |
 | `health.readiness.verifyDependencies`  | Indication if readiness probes should verify if Promitor can interat with external dependencies. Do note that this will contact all dependencies which can have performance impact, cause throttling or cascading failures when consumed very often. | `false`            |   |
 | `health.readiness.delay`  | Amount of seconds to wait before probing the container to verify if it's ready | `5`            |   |
+| `health.readiness.failureThreshold`  | Number of retries after first failed check before container will be marked `Unready`  | `3` |
```

Comment:
```suggestion
| `health.readiness.thresholds.failure`  | Number of retries after first failed check before container will be marked `Unready`  | `3` |
```

Reply:
Sorry missed another commit. Now it should be intact.